x/incentive/types/params_test.go

```diff
@@ -0,0 +1,152 @@
+package types_test
+
+import (
+	"testing"
+	"time"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/kava-labs/kava/x/incentive/types"
+	"github.com/stretchr/testify/suite"
+)
+
+type paramTest struct {
+	params     types.Params
+	expectPass bool
+}
+
+type ParamTestSuite struct {
+	suite.Suite
+
+	tests []paramTest
+}
+
+func (suite *ParamTestSuite) SetupTest() {
```

Comment:
Recommend a table test here. Not a huge fan of testing with `p1, p2, ...` as it separates `expectPass` from the params.